pkg/lib/manifest2po.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
+
+# Extracts translatable strings from manifest.json files.
+
+import argparse
+import json
+import os
+import re
+import sys
+
+import polib
+
+parser = argparse.ArgumentParser(
```

Comment:
Can you intent all of the top-level stuff like:

```python
import ..

def main() -> None:
    ( argparse, etc, etc. )

if __name__ == '__main__':
    main()
```

?

That makes it possible to import the file without executing it.

My personal preference is to see main() as the last thing in the file except for its own invocation.